arviz/tests/base_tests/test_data.py

```diff
@@ -873,3 +873,41 @@ def test_id_conversion_args(self):
         assert isinstance(inference_data, InferenceData)
         assert set(inference_data.posterior.coords["Ivies"].values) == set(IVIES)
         assert inference_data.posterior["theta"].dims == ("chain", "draw", "Ivies")
+
+
+
+class TestDataArrayToDataset:
+    def test_1d_dataset(self):
+        size = 100
+        dataset = convert_to_dataset(xr.DataArray(np.random.randn(1, size),
+                                                  name="plot", dims=('chain', 'draw')))
+        assert len(dataset.data_vars) == 1
+        assert dataset.variables.get("plot") is not None
```

Comment:
how about `assert "plot" in dataset.data_vars`?

Reply:
Ok, I will make changes. 
But, where can we see code style issues by black? Because I did not see any suggestion in automation check that runs after each commit.

Reply:
you can run `black --check` I believe and it will run black checks. The CI will run black on every commit that is pushed so you'll see a failure on azure pipelines, but its easier just to check locally

Reply:
Ah nevermind, what I said is how things should ideally run. I saw oriols comment below that CI is not failing on black errors right now

Reply:
Yes it can easily run locally. I will run all checks before pushing any changes.